misc/scripts/deploy-tools.sh

```diff
@@ -193,23 +192,16 @@ function cleanvm {
     done
 }
 
-function cleanupVolumes {
-    log "cleanupVolumes: Asking docker $TARGET to remove volumes ($VOLUMES)"
-    for vol in $VOLUMES
-    do
-        $SSH $TARGET "if docker volume ls | $GREP -q $vol; then \
-        docker volume rm $vol; fi "
-    done
-}
-
 function cleanupVMPre {
     $SSH $TARGET "$IS_SYSTEMD"
     if [ $? -eq 0 ]
     then
         $SSH $TARGET systemctl stop $PLUGIN_NAME
         $SSH $TARGET $PIDOF $PLUGIN_NAME
+	$SSH $TARGET systemctl restart docker
```

Comment:
Will do.. the Mac vim was not setup.. and I have not yet gotten to see why my dev box is down..